boards/openmote-b/include/board.h

```diff
@@ -88,6 +88,9 @@
 
 /**
  * @name    OpenWSN leds configuration
+ *
+ * @note This configuration mimics the one done in OpenWSN-fw for the
+ *       same platform
```

Comment:
Wait, this should use `cpu/cc2538/include/openwsn_defs.h`

Reply:
I kept ledpins and debugpins here, since they are completely board specific, do you think it should be put in its own file as well?

Reply:
ah sorry didn't see this was about the LEDs